src/status_im/contexts/wallet/common/activity_tab/view.cljs

```diff
@@ -26,7 +26,8 @@
                           :collectible-name   (if (> amount 1)
                                                 (str amount " " nft-name)
                                                 nft-name)
-                          :collectible-number token-id}
+                          :collectible-number (when (not= token-id "0")
+                                                token-id)}
```

Comment:
Status-go replies with a token-id `0x0` for collectibles, seems this is just a default number always comming, not exactly the collectible number, so I'm skipping that case